src/components/Books.js

```diff
@@ -1,16 +1,33 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
-import BookList from './ListBooks';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchBooks } from '../redux/Books/books';
+import Book from './Book';
 import Forms from './form';
 
-const Books = () => {
-  const books = useSelector((state) => state.book);
+function Books() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchBooks());
+    // eslint-disable-next-line
+    }, []);
+
+  const books = useSelector((state) => state.handleBook.entities);
+
   return (
     <>
-      <BookList books={books} />
+      <div className="book-mark">
```

Comment:
- [ ] @Ngufor-emmanuella great work in your design but some elements don't match the zeplin design, kindly check the comparison below and ensure your app stying matches the one given:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/202776316-84eb73bc-076d-48f7-b656-40a664fab117.png) | ![image](https://user-images.githubusercontent.com/61408860/205399050-4cfb6c36-cc11-4456-93a1-9c730f69757b.png)